pages/index.js

```diff
@@ -51,3 +58,13 @@ export default function Home() {
     </div>
   )
 }
+
+Home.getInitialProps = async function() {
```

Comment:
This is not wrong but you have to implement the frontend only solution for first phase

Reply:
How is the front-end only solution different ? example ?

Reply:
This code is executed on the server side when your code is compiled. This does not execute on the browser. You can try it with adding console to it.
I want the implementation which will work on Browser only and no server side rendering.

Reply:
https://www.google.com/search?q=react+fetch+data&rlz=1C5CHFA_enIN741IN741&oq=react+fetch+data&aqs=chrome.0.69i59j0l6j69i60.3230j0j7&sourceid=chrome&ie=UTF-8
https://reactjs.org/docs/faq-ajax.html